src/archiefvernietigingscomponent/js/components/destruction-admin/index.js

```diff
@@ -0,0 +1,44 @@
+import React, {useState} from "react";
+import ReactDOM from "react-dom";
+import {jsonScriptToVar} from "../../utils";
+import {ZaaktypeSelect} from "../destruction-list-create/zaaktype-select";
+
+
+const SelectShortReviewZaaktypen = ({zaaktypeChoices, initialZaaktypes}) => {
+    const [selectedZaaktypes, setSelectedZaaktypes] = useState(initialZaaktypes);
+
+    return (
+        <div className="form-row field-short_review_zaaktypes">
+            <label htmlFor={"id_short_review_zaaktypes"}>Short review zaaktypen:</label>
+            <ZaaktypeSelect
+                zaaktypen={zaaktypeChoices}
+                selectedZaaktypen={selectedZaaktypes}
+                setSelectedZaaktypen={(selected) => {
+                    setSelectedZaaktypes(selected);
+                }}
+                checkboxName='short_review_zaaktypes'
+            />
+        </div>
+    );
+};
+
+
+const mount = () => {
+    const nodes = document.getElementsByClassName('field-short_review_zaaktypes');
+    if (nodes.length === 0) return;
+
+    const zaaktypeChoices = jsonScriptToVar('zaaktype-choices');
+    const initialZaaktypes = jsonScriptToVar('initial-zaaktypes');
+
+    const node = nodes[0];
+
+    ReactDOM.render(
+        <SelectShortReviewZaaktypen
+            zaaktypeChoices={zaaktypeChoices}
```

Comment:
Does this show a bunch of URLs in the admin?

Reply:
![Screenshot from 2021-03-10 17-01-19](https://user-images.githubusercontent.com/19154114/110658839-5420b400-81c2-11eb-930a-c2244f5073a1.png)

I tried making a user-friendly front end for it

Reply:
Interesting, I couldn't read this in the JS. Looks good!